acceptance/acceptance_test.go

```diff
@@ -853,7 +853,37 @@ func testAcceptance(
 
 								assertOutput := assertions.NewLifecycleOutputAssertionManager(t, output)
 								assertOutput.IncludesLifecycleImageTag(lifecycle.Image())
-								assertOutput.IncludesSeparatePhasesWithExtension()
+								assertOutput.IncludesSeparatePhasesWithBuildExtension()
+
+								t.Log("inspecting image")
+								inspectCmd := "inspect"
+								if !pack.Supports("inspect") {
+									inspectCmd = "inspect-image"
+								}
+
+								output = pack.RunSuccessfully(inspectCmd, repoName)
+							})
+						})
+
+						when("there are run image extensions", func() {
```

Comment:
This test won't run in CI until there's a lifecycle that supports run image extension. But I've verified with my dev lifecycle that it does the job.